parastell/invessel_build.py

```diff
@@ -309,6 +309,23 @@ def export_step(self, export_dir=""):
             )
             cq.exporters.export(component, str(export_path))
 
+    def export_stl(self, export_dir=""):
```

Comment:
Since this function is functionally identical to `export_step` it might be better to rename `export_step` as `export_standard` (or something) and add the export type as an argument.

Reply:
You could change `export_step` to `export_cad` and have the function take the file extension as an argument (in string format)